symtabAPI/h/relocationEntry.h

```diff
@@ -39,6 +39,20 @@
 #include "Annotatable.h"
 #include "Region.h"
 
+// The minimum version of libelf Dyninst supports (0.173) does not define:
```

Comment:
Dyninst [requires](https://github.com/dyninst/dyninst/blob/v13.0.0/cmake/tpls/DyninstElfUtils.cmake#L29) 0.186 or newer. We could increase that (0.95 is coming out next month), as long as it wouldn't need to be a really recent version. Do you happen to know when those constants were added?

Reply:
Hmm. 0.190 was released in 2023. That might be "too new" for some folks. @kupsch What do you think about bumping the min version?

Reply:
>Probably too new. I think defining the couple of macros if not defined is OK until we do update.

That works for me.

> It probably wouldn't hurt to put a comment in tpls/DyninstElfUtils.cmake (above the minimum version): delete the RISCV elf macros if version upgraded >= 0.190

@wxrdnx Could you add this?